src/components/CheckoutCard.vue

```diff
@@ -183,19 +183,19 @@ export default class CheckoutCard<
     }
 
     protected showSuccessScreen() {
-        this.statusScreenTitle = 'Payment successful';
+        this.statusScreenTitle = this.$t('Payment successful') as string;
         this.statusScreenState = StatusScreen.State.SUCCESS;
         this.showStatusScreen = true;
         window.setTimeout(() => this.$rpc.resolve({success: true}),  StatusScreen.SUCCESS_REDIRECT_DELAY);
     }
 
     protected showUnderpaidWarningScreen() {
         this.paymentState = PaymentState.UNDERPAID;
-        this.statusScreenTitle = 'Incomplete payment';
-        this.statusScreenMessage = 'You transferred an insufficient amount. ' +
-            'Please contact the merchant for a refund. Restart to pay for your order.';
+        this.statusScreenTitle = this.$t('Incomplete payment') as string;
+        this.statusScreenMessage = this.$t('You transferred an insufficient amount. ') as string +
+            this.$t('Please contact the merchant for a refund. Restart to pay for your order.') as string;
```

Comment:
You mean like this: 
```
const statusScreenMessage = 'You transferred an insufficient amount. ' +
    'Please contact the merchant for a refund. Restart to pay for your order.';
this.statusScreenMessage = this.$t(statusScreenMessage) as string;

```
?

Reply:
Mh, ok you merged the text but now it's a very long line.
I meant concatenating them with `+` within the `$t`.

Reply:
It's unfortunately not possible, since i'm extracting what's between the $t parenthesis for the translation file. If i add something that's not a string (like a +), then it just break.
But if you really want to support concatenation inside the $t function, i can add support for it in the webpack-i18n-tool extract script. Just let me know.
Furthermore, i'll write details on how the i18n is working for the hub & the wallet later this week.